artview/components/menu.py

```diff
@@ -264,13 +268,13 @@ def _about(self):
         txOut = "This is a simple radar file browser to allow \
                  quicklooks using the DoE PyArt software"
         QtGui.QMessageBox.about(self, "About ARTView", txOut)
- 
+
     def _get_RadarLongInfo(self):
-        '''Print out the radar info to text box.'''
+        '''Print out the radar info to text box and terminal.'''
```

Comment:
I have to admit this is REALLY low on my to-do. So it's on my list but I won't get to this anytime soon.